### DISCUSSION TITLE
Centre image vertical and horizontal

### DISCUSSION THREAD

**Discussion by jhanley87** [twilio-labs/paste#2731] [General] [answered]
Hey all I would like to have a box that is 100% height and an image in the middle (vertical and horizontal) current code i have is:
```
<Box style={{minHeight:"100%"}}>
--- what goes here? ---
</Box>
```
example of what I am looking for here:
![image](https://user-images.githubusercontent.com/96482686/195384989-a0230999-2ee4-488e-aeb2-493e8c661337.png)

**Comment by nkrantz** [comment, marked as answer]
Hey @jhanley87 - welcome to Paste! I'd recommend doing something like this:

```
<Box display="flex" justifyContent="center" alignItems="center">
  {your img here}
</Box>
```

In general, it's best to use the properties directly on the Box component rather than putting them in the `style` tag. You can see [the list of Box Props on the docs site](https://paste.twilio.design/primitives/box#props).

Let us know how it goes!

**Reply by jhanley87** [reply]
That is perfect, thanks... also switched to the properties, thanks for the heads up
